core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/CaseSensitivitySpec.scala

```diff
@@ -21,6 +21,8 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.time.{Millis, Span}
 
 import scala.concurrent.ExecutionContext
+import org.broadinstitute.dsde.rawls.model.AttributeName.toDelimitedName
+import slick.jdbc.MySQLProfile.api._
```

Comment:
the Rawls convention is to `import driver.api._` as needed, at the class level e.g. https://github.com/broadinstitute/rawls/blob/f704fc8e693f0c14bcaba78d9e9b7b53e77daa7c/core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccessSpec.scala#L11
or even for an individual test, e.g.
https://github.com/broadinstitute/rawls/blob/b844baa8e6179489dccfe5d6046ca28736c68821/core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/CaseSensitivitySpec.scala#L378

I suspect this is functionally the same as your import - and I honestly don't know if one is better than the other - but suggest sticking with the convention for consistency

Reply:
Fixed! Moved this to the test level-